src/document-submit.js

```diff
@@ -131,6 +131,7 @@ export function onDocumentFormSubmit_(e) {
     e.stopImmediatePropagation();
 
     const actions = Services.actionServiceForDoc(form);
-    actions.execute(form, 'submit', /*args*/ null, form, e, ActionTrust.HIGH);
+    actions.execute(
+        form, 'submit', /*args*/ null, form, form, e, ActionTrust.HIGH);
```

Comment:
Let's add `/*source*/` and `/*caller*/` as appropriate here.

Reply:
Done.